<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="thundurus" failures="1" tests="3" name="Test" time="13.194" errors="0" timestamp="27 Jul 2019 20:18:09 GMT">
  <testcase classname="testCases.searchPg" name="search" time="4.949"/>
  <testcase classname="testCases.listingPg" name="listing" time="4.02"/>
  <testcase classname="testCases.productPg" name="product" time="4.15">
    <failure type="java.lang.IllegalArgumentException" message="bound must be positive">
      <![CDATA[java.lang.IllegalArgumentException: bound must be positive
at java.base/java.util.Random.nextInt(Random.java:388)
at pageModels.product_pg.swatches(product_pg.java:33)
at testCases.productPg.product(productPg.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- product -->
  <testcase classname="generic.openBrowser" name="@BeforeSuite open" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite open -->
  <testcase classname="generic.openBrowser" name="@BeforeSuite open" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite open -->
  <testcase classname="generic.openBrowser" name="@AfterSuite close" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite close -->
  <testcase classname="generic.openBrowser" name="@AfterSuite close" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite close -->
</testsuite> <!-- Test -->
